js/src/lib/interfaces/Libraries.ts

```diff
@@ -233,14 +233,22 @@ model = timm.create_model("hf_hub:${model.id}", pretrained=True)`;
 const sklearn = (model: ModelData) => {
 	if (model.tags?.includes("skops")) {
 		const skopsmodelFile = model.config?.sklearn?.filename;
+		const skopssaveFormat = model.config?.sklearn?.save_format;
+		if (skopssaveFormat == "pickle")
+		{
+			return `download("${model.id}", "path_to_folder")
+import joblib
+model = joblib.load(
+	"${skopsmodelFile}"
+)`
+} else {
 		return `from skops.hub_utils import download
 from skops.io import load
 
 download("${model.id}", "path_to_folder")
-# make sure model file is in skops format
-# if model is a pickle file, make sure it's from a source you trust
```

Comment:
Should we still have this warning but in the pickle snippet?

Reply:
I just removed it, was a merge conflict